corehq/apps/app_manager/models.py

```diff
@@ -5246,6 +5246,9 @@ class Application(ApplicationBase, TranslationMixin, HQMediaMixin):
     grid_form_menus = StringProperty(default='none',
                                      choices=['none', 'all', 'some'])
 
+    def has_modules(self):
+        return len(self.modules) > 0
```

Comment:
Might as well have a `not self.is_remote_app()` check here. (I just got bitten by this.)